config/clusters/farallon/common.values.yaml

```diff
@@ -90,6 +90,8 @@ basehub:
         type: LoadBalancer
       https:
         enabled: true
+        letsencrypt:
+          contactEmail: yuvipanda@gmail.com
```

Comment:
I wonder if we should change this to `support@2i2c.org` to remove you as a bottle neck?

Reply:
@sgibson91 we definitely should but I didn't want to do that as part of this change!  As we move these two clusters to using support chart (#1278) this should just go away.